frontend/libs/studio-feedback-form/src/utils/questionUtils.ts

```diff
@@ -0,0 +1,18 @@
+import type { QuestionConfig } from '../types/QuestionsProps';
+
+type QuestionType = Extract<QuestionConfig, { type: string }>['type'];
+
+const defaultValueMap: Record<QuestionType, any> = {
```

Comment:
Would this be a little more type-safe? 👼 
```suggestion
const defaultValueMap: Record<QuestionType, string | Array<any> = {
```